monai/bundle/scripts.py

```diff
@@ -1619,3 +1619,42 @@ def create_workflow(
     workflow_.initialize()
 
     return workflow_
+
+
+def download_large_files(bundle_path: str, large_file_name: str | None = None) -> None:
```

Comment:
I think the bundle path could be 'os.getcwd()' if it's not specified?

Reply:
Good point, have updated it in the latest commit. Thanks!